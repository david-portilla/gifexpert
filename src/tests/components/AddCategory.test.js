import React, { useState } from 'react'
const { shallow } = require('enzyme')
const { AddCategory } = require('../../components/AddCategory')

describe('Test <AddCategory />', () => {

  const setCategories = () => { }

  test('should recive a function', async () => {
    const wrapper = shallow(<AddCategory setCategories={ setCategories } />)
    expect(wrapper).toMatchSnapshot()
  })
})