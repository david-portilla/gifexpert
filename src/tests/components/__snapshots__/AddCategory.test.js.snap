// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <AddCategory /> should recive a function 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    onClick={[Function]}
    type="text"
    value=""
  />
</form>
`;
